lib/syskit/test/profile_assertions.rb

```diff
@@ -465,6 +434,62 @@ def assert_can_deploy_together(*actions)
                       e.message, e.backtrace
             end
 
+            # Tests that the following syskit-generated actions can be ALL deployed at the
+            # same time, that is they result in a valid, non-abstract network whose all
+            # components have a deployment
+            #
+            # When resolving actions that are not directly defined from profile
+            # definitions, the method will attempte to resolve method action by
+            # calling them. If there is a problem, pass the action model to the
+            # `exclude` argument.
+            #
+            # In particular, in the presence of action methods with required
+            # arguments, run one assert first with the action method excluded and
+            # another with that action and sample arguments.
+            #
+            # @param action_or_profile if an action interface or profile, test all
+            #   definitions that are reachable from it. In the case of action interfaces,
+            #   this means looking into method actions and action state machines.
+            # @param together_with test that all actions in `action_or_profile`
+            #   can be instanciated when all actions in `together_with` are instanciated
+            #   at the same time. This can be used if the former depend on the presence
+            #   of the latter, or if you want to test against conflicts.
+            def assert_can_deploy_all(
+                action_or_profile = subject_syskit_model,
+                exclude: [], together_with: []
+            )
+                actions = assert_actions(action_or_profile, exclude: exclude)
+
+                together_with = assert_together_with(together_with, exclude: exclude)
+
+                assert_can_deploy_all_together(actions, *together_with)
+            end
+
+            # Spec-style call for {#assert_can_deploy_all}
+            #
+            # @example verify that each definition of a profile can be deployed
+            #   describe MyBundle::Profiles::MyProfile do
+            #     it { can_deploy_all }
+            #   end
+            def can_deploy_all(
+                action_or_profile = subject_syskit_model, together_with: []
+            )
+                assert_can_deploy_all(action_or_profile, together_with: together_with)
+            end
+
+            # Tests that the given syskit-generated actions can be ALL deployed together
+            #
+            # It is stronger (and therefore includes)
+            # {assert_can_deploy_together}
+            def assert_can_deploy_all_together(*actions)
+                syskit_run_deploy_in_bulk(
+                    actions.flatten, compute_policies: true, compute_deployments: true
+                )
+            rescue Minitest::Assertion, StandardError => e
+                raise ProfileAssertionFailed.new("deploy all together", actions, e),
+                      e.message, e.backtrace
+            end
+
```

Comment:
assert_can_deploy_together is already doing the job, I don't see the need for this one. Just flatten and splat the `actions` arguments in the assert methods.